From d3c4b52933ab2dc97e2c57e6d1a6d8252ab0beb4 Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Thu, 26 May 2011 17:18:03 +0800
Subject: [PATCH 302/691] ENGR00144060 MX53_SMD: recovery android, update
 recovery command for emmc.

Change the recovery boot for MX53_SMD to emmc 's device 1.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 include/configs/mx53_smd_android.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/configs/mx53_smd_android.h b/include/configs/mx53_smd_android.h
index 04fc808..f982cdc 100644
--- a/include/configs/mx53_smd_android.h
+++ b/include/configs/mx53_smd_android.h
@@ -104,7 +104,7 @@
 	"setenv bootargs ${bootargs} init=/init root=/dev/mmcblk0p4 rootfs=ext4 video=mxcdi1fb:RGB666,XGA ldb=di1 di1_primary"
 #define CONFIG_ANDROID_RECOVERY_BOOTCMD_MMC  \
 	"run bootargs_base bootargs_android_recovery;"  \
-	"mmc read 0 ${loadaddr} 0x800 0x2000;bootm"
+	"mmc read 1 ${loadaddr} 0x800 0x2000;bootm"
 #define CONFIG_ANDROID_RECOVERY_CMD_FILE "/recovery/command"
 
 #define CONFIG_ANDROID_SYSTEM_PARTITION_MMC 2
-- 
1.7.7.4

